report/_toc.yml

```diff
@@ -25,6 +25,7 @@ parts:
     chapters:
       - file: validation/plasma-driven-permeation/plasma-driven-permeation
       - file: validation/thermodesorption_spectra/ogorodnikova/ogorodnikova-tds
```

Comment:
I think we should change the name of the Ogorodnikova tungsten TDS instead of just "Thermo desorption spectrum"

Reply:
Sure, I'll add it as an issue.

Reply:
I think you can do it in this PR

Reply:
Changed it in #30 .